### DISCUSSION TITLE
[Feature Request] Support `disableEnforceFocus` in Modal Component

### DISCUSSION THREAD

**Discussion by spencer-hivert** [heroui-inc/heroui#4750] [Feature requests]
### Is your feature request related to a problem? Please describe.

We present the [react-plaid-link](https://github.com/plaid/react-plaid-link) from the `Modal` component, and are facing an issue similar to the one discussed [here](https://github.com/plaid/react-plaid-link/issues/160).

Plaid keeps flickering, preventing the user from entering their credentials. Upon further inspection, the NextUI modal is preventing focus from leaving while it's open.

### Describe the solution you'd like

To solve this problem, I'd like to see an `disableEnforceFocus` prop which behaves similarly to the behavior described in the [Material UI Modal component](https://mui.com/material-ui/api/modal/#modal-prop-disableEnforceFocus).

### Describe alternatives you've considered

As a workaround, we've been forced to modify how we present the Plaid experience, resulting in an inferior user experience.

**Comment by linear** [comment]
<p><a href="https://linear.app/nextui-inc/issue/ENG-1107/[feature-request]-support-disableenforcefocus-in-modal-component">ENG-1107 [Feature Request] Support `disableEnforceFocus` in Modal Component</a></p>

**Comment by spencer-hivert** [comment]
Following up on this issue - are there any suggestions for a workaround?
